clients/src/test/java/org/apache/kafka/common/security/oauthbearer/OAuthBearerValidatorCallbackHandlerTest.java

```diff
@@ -81,6 +84,52 @@ public void testInvalidAccessToken() throws Exception {
         assertInvalidAccessTokenFails(createAccessKey("{}", "{}", "{}"), substring);
     }
 
+    @Test
+    public void testConfigureThrowsExceptionOnJwtValidatorInit() throws IOException {
+        OAuthBearerLoginCallbackHandler handler = new OAuthBearerLoginCallbackHandler();
+        JwtRetriever jwtRetriever = new JwtRetriever() {
+            @Override
+            public void init() throws IOException {
+                throw new IOException("My init had an error!");
+            }
+
+            @Override
+            public String retrieve() {
+                return "dummy";
+            }
+        };
+
+        Map<String, ?> configs = getSaslConfigs();
+
+        try (JwtValidator jwtValidator = new DefaultJwtValidator(configs, OAuthBearerLoginModule.OAUTHBEARER_MECHANISM)) {
+            assertThrowsWithMessage(
+                KafkaException.class, () -> handler.init(jwtRetriever, jwtValidator), "encountered an error when initializing");
+        }
+    }
+
+    @Test
+    public void testConfigureThrowsExceptionOnJwtValidatorClose() throws IOException {
+        OAuthBearerLoginCallbackHandler handler = new OAuthBearerLoginCallbackHandler();
+        JwtRetriever jwtRetriever = new JwtRetriever() {
+            @Override
+            public void close() throws IOException {
+                throw new IOException("My close had an error!");
+            }
+            @Override
+            public String retrieve() {
+                return "dummy";
+            }
+        };
+
+        Map<String, ?> configs = getSaslConfigs();
+        try (JwtValidator jwtValidator = new DefaultJwtValidator(configs, OAuthBearerLoginModule.OAUTHBEARER_MECHANISM)) {
+            handler.init(jwtRetriever, jwtValidator);
+
+            // Basically asserting this doesn't throw an exception :(
+            handler.close();
```

Comment:
How about use the `assertDoesNotThrow`
```suggestion
            assertDoesNotThrow(handler::close);
```